src/drivers/intel/cavs/dmic.c

```diff
@@ -628,7 +628,7 @@ static int select_mode(struct dai *dai,
 	/* Calculate remaining gain to FIR in Q format used for gain
```

Comment:
Typo in commit message: Should be "Change" instead of "Challenge".

Reply:
Okay I will change it.

Reply:
Not all commits were fixed.

Reply:
Since I didn't mention "challenge" in every patch's log, that is why I needed to change only once.

Reply:
You originally had two commits with that word. I drew your attention that you didn't fix both.

Reply:
The comment which still has the wrong word is "src: drivers: Remove unnecessary parentheses". Please recheck it.

Reply:
Got it, I will change it.